.github/ISSUE_TEMPLATE/task.yaml

```
@@ -1,5 +1,5 @@
 name: Task
-description: File an enhancement proposal
+description: File an enhancement proposal (e.g., feature requests or documentation improvements)
```

https://docs.ubuntu.com/styleguide/en#latin-words-and-phrases
```suggestion
description: File an enhancement proposal (such as feature requests or documentation improvements)
```